src/libUtils/JsonUtils.h

```diff
@@ -23,6 +23,8 @@
 #include <sstream>
 #include <string>
 
+#include "libUtils/Logger.h"
```

Comment:
I think this `#include` should move elsewhere.  (IWYU principle)

Reply:
The inline code uses the logger, this was added so the json utils could be used on its own without having to pre include heade files in your own source